app/views/usage/index.html.erb

```diff
@@ -10,7 +10,7 @@
            locals: { user_count: @total_org_users,
                      plan_count: @total_org_plans,
                      separators: @separators } %>
-<% if @funder.present? %>
+<% if !current_user.can_org_admin? && !current_user.can_super_admin? %>
   <%= render partial: 'usage/template_statistics' %>
 <% else %>
```

Comment:
I think this piece of code would also allow org admins to see the link to the template statistics from all the system. If that is the case then we should avoid it.

You can look at the partials `template_statistics` nad `plans_by_template_chart`

The last template uses the method `prep_data_for_template_plans_chart` .